public/app/features/transformers/docs/content.ts

```diff
@@ -63,7 +63,7 @@ export const transformationDocsContent: TransformationDocsContentType = {
   ${buildImageContent(
     '/static/img/docs/transformations/add-field-from-calc-stat-example-7-0.png',
     imageRenderType,
-    this.name
```

Comment:
`this` is not bound in the correct context when the `buildImageContent` function is being called, so name is `undefined`.